invenio_records_rest/__init__.py

```diff
@@ -317,7 +317,26 @@
 
 
 A custom query parser can also be plugged in by setting the ``search_factory``
```

Comment:
Is not on the changes of the commit but the exact name of the setting is ``search_factory_imp`` (is missing the "_imp" part)